webpack/webpack.common.js

```diff
@@ -49,4 +49,11 @@ module.exports = {
       ],
     }),
   ],
+  devServer: {
+    host: '0.0.0.0',
+    disableHostCheck: true,
+    watchOptions: {
+      poll: true // Or you can set a value in milliseconds.
+    }
+  }
```

Comment:
@devpurohit please add this to the [webpack.dev.js](https://github.com/sydrawat/react-boilerplate/blob/master/webpack/webpack.dev.js) file at [line 33](https://github.com/sydrawat/react-boilerplate/blob/master/webpack/webpack.dev.js#L33).

Reply:
Will change it.